source/index.html

```diff
@@ -1,39 +1,282 @@
+                    <a href="mailto:poetspetparlor@gmail.com"><img src="./assets/email.svg"></a>
+                </div>
+                <p class="philosophy">Poets pet parlor provides the only 24 hour staffed care for your pet in Marquette County. Offering full service grooming, doggie daycare and luxury boarding, in a
+                safe, fun, and homelike environment. At poets pet parlor, we treat your pet like our own.</p>
+            </div>
+        </header>
+        <section id="about-me">
+            <img class="dog-divider" src="./assets/new-dog1.png">
+            <hr>
+            <div class="content">
+                <h2>ABOUT ME &amp POET</h2>
+                <p>I am the proud owner of 4 mix breed dogs, Oli, Saige, Lucy and Sunny, two macaws, and four cats whom I love with all her heart. They are my family, and companions, and my inspiration for Starting Poets Pet Parlor. I began as a dog groomer who wanted to offer cage free options for grooming and as my clientele grew, I was asked to watch and care for many of my clients pets who did not want to leave their pets at a traditional kennel. I expanded my business to include doggie daycare and now pet boarding that is aimed to provide the most homelike environment for your dog as possible. As an alternative to the traditional dog kennel, I have created a safe and fun environment where your dog can romp with friends in indoor and outdoor play yards, sleep in a home-like bedroom, or even sleep in an on-site apartment in bed with staff.</p>
+                <p>Poet was my first dog and namesake for Poets Pet Parlor. He was an amazing dog who began my love for animals. He passed away 2 months after the opening of Poets.</p>
+            </div>
+        </section>
+        <section id="clients">
+            <img class="dog-divider" src="./assets/new-dog5.png">
+            <hr>
+            <div class="content">
+                <h2>OUR CLIENTS</h2>
+                <div id="instafeed"></div>
+            </div>
+        </section>
+        <section id="grooming">
+            <img class="dog-divider" src="./assets/new-dog2.png">
+            <hr>
+            <div class="content">
+                <h2>GROOMING SERVICES</h2>
+                <div class="list-info">
+                    <div class="primary">
+                        <h3>Full Service Haircut</h3>
+                        <p>Includes nail trim, ear cleaning, anal gland expression, bath, blowdry and hair cut.
+                            <br>- price varies by bread and size, please call for pricing.</p>
+                        <h3>Bath</h3>
+                        <p>Includes nails, gland expression, ear cleaning, bath, blowdry and brushout.</p>
+                    </div>
+                    <aside>
+                        <h3>Additonal Services</h3>
+                        <ul>
+                            <li>Nails - $5</li>
+                            <li>Anal Glands (external expression) - $10</li>
```

Comment:
"Anal glands" (I think sentence-case works well here, and should be consistent … I just mean, all the other items in the list have only the first letter capitalized).